tests/.pug-lintrc

```diff
@@ -0,0 +1,19 @@
+{
+"disallowAttributeInterpolation": true,
+"disallowClassAttributeWithStaticValue": true,
+"disallowDuplicateAttributes": true,
+"disallowHtmlText": true,
+"disallowIdAttributeWithStaticValue": true,
+"disallowLegacyMixinCall": true,
+"disallowMultipleLineBreaks": true,
+"disallowSpacesInsideAttributeBrackets": true,
+"requireClassLiteralsBeforeAttributes": true,
+"requireIdLiteralsBeforeAttributes": true,
+"requireLowerCaseAttributes": true,
+"requireLowerCaseTags": true,
+"requireSpaceAfterCodeOperator": true,
+"requireStrictEqualityOperators": true,
+"validateAttributeSeparator": { "separator": ", ", "multiLineSeparator": ",\n    " },
+"validateDivTags": true,
+"validateTemplateString": true
+}
```

Comment:
Moving it to the root would bring us closer to balancing the number of linting configs we have in `/` and `/tests`.